homeassistant/components/smarty/sensor.py

```diff
@@ -0,0 +1,230 @@
+"""Support for Salda Smarty XP/XV Ventilation Unit Sensors."""
+
+import datetime as dt
+import logging
+
+from homeassistant.core import callback
+from homeassistant.components.binary_sensor import BinarySensorDevice
```

Comment:
The binary sensor entities should move to a binary sensor platform module.

Reply:
Hi Martin, it seems that I've messed up some commits...
Somebody on discord suggested that I close the PR and start fresh.
Should I do that or try to fix this?

Reply:
Either try to rebase and remove the unrelated commits, if you're comfortable with that, or make a new PR and link to this one.